src/app/services/model/model.service.ts

```diff
@@ -10,10 +10,10 @@ export class ModelService {
     constructor(private _http: HttpClient) { }
 
     submitSurfaceRequest( params: ISurfaceParameters ) {
-        return this._http.get(environment.msisApi + '/surface', { params: params as any } );
+        return this._http.get(environment.msisApi + 'surface', { params: params as any } );
```

Comment:
In the env file you had the `/` with the other urls, so I changed this to be consistent with those other URLs. Happy to change back to this if that would be better.

Reply:
I honestly forget what the standard is…this looks right.

Reply:
Actually, it really shouldn't be handled with strings at all it looks like. I think if you put `URL('surface', environment.msisApi)` it would handle either case for us without failing...

https://nodejs.org/api/url.html#new-urlinput-base